lib/board.rb

```diff
@@ -1,23 +1,37 @@
 class Board
-  attr_reader :board, :moves_remaining
+  attr_reader :squares
 
-  def initialize(board)
-    @board = board
-    @moves_remaining = 9
+  def initialize(squares)
+    @squares = squares
+    @winning_combinations = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [2, 4, 6],
+      [0, 4, 8]
+    ]
   end
 
   def player_make_move(player_mark, position)
-    @board[position - 1] = player_mark
-    update_moves_remaining
+    @squares[position - 1] = player_mark
   end
 
-  def position_available?(position)
-    @board[position - 1].is_a? Integer
+  def move_valid?(move)
+    move = move.to_i
+    (@squares[move - 1].is_a? Integer) && move.between?(1, 9)
```

Comment:
Nice! But I would reverse the order, as `&&` will only evaluate the second statement if the first is true. So if move is not between 1-9 then you don't need to do the integer check

Reply:
Good point! Thanks, I'll reverse the order